src/main/java/org/folio/bulkops/domain/bean/ExtendedHoldingsRecord.java

```diff
@@ -8,8 +8,10 @@
 import lombok.NoArgsConstructor;
 import lombok.With;
 import org.folio.bulkops.domain.dto.IdentifierType;
+import org.folio.bulkops.domain.dto.TenantNotePair;
```

Comment:
TenantNotePair can be renamed to ExtendedNoteTypeName o follow general rule when including tenant information.